ocaml/idl/datamodel.ml

```diff
@@ -1421,7 +1422,12 @@ let _ =
 
   error Api_errors.pvs_server_address_in_use ["address"]
     ~doc:"The address specified is already in use by an existing PVS_server object"
-    ()
+    ();
+
+  (*Feature errors*)
+  error Api_errors.multiple_feature_instance ["feature"; "host"]
```

Comment:
For this one, the case is that there are two records for the same feature for the same host? All we need is to make the DB reflect the info xapi gets from v6d. So, we just need to delete one record and update the other.